src/vmspawn/vmspawn.c

```diff
@@ -105,6 +106,10 @@ static int help(void) {
                "                            Configure QEMU's networking stack\n"
                "     --secure-boot=BOOL     Configure whether to search for firmware which\n"
                "                            supports Secure Boot\n\n"
+               "%3$sUser Namespacing:%4$s\n"
+               "     --private-users=UIDBASE[:NUIDS]\n"
+               "                            Configure the UID/GID range to map into the\n"
+               "                            virtiofsd namespace\n\n"
```

Comment:
i am not sure i grok this. what dos this do?

Reply:
Currently mkosi builds images inside of a user namespace, which means the resultant directory image is owned by PID 0 in the user's subuid namespace.
In order to boot an image like this I need to tell virtiofsd to map the external user namespace UIDs into the correct UIDs for the VM.
While discussing with Daan he asked that I call this --private-users. I agree that the help message isn't very helpful though.
The man page has a much more thorough explanation and gives a hint on when to use it under the part for `-D`